source/connect/tls-ssl-connection.txt

```diff
@@ -11,3 +11,86 @@ TLS/SSL Connection Tab
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`TLS / SSL` tab allows you to connect deployments via TLS / SSL.
+For more information on :guilabel:`TLS / SSL`, see :manual:`TLS Options 
+</reference/connection-string/#tls-options>`
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`TLS / SSL` tab.
+
+      You have the option to use a ``Default`` connection or to set the TLS / SSL 
+      connection ``On`` or ``Off``. 
+
+      Default
```

Comment:
Just curious, could we maybe try putting these headings into a table with one column for the option (`Default`, `On`, `Off`) and one column that describes the behavior for that option?

Reply:
Done.